templates/experiments/components/versions/compare.html

```diff
@@ -0,0 +1,16 @@
+<div class="my-2">
+    {% if version_details.fields_changed %}
+        {% for field in version_details.fields %}
+            <div class="grid grid-cols-2 gap-2 max-h-64 overflow-y-auto">
+                {% include 'experiments/components/versions/version_field_details.html' with field=field.previous_field_version field_changed=field.changed diff_color='bg-red-400' %}
+                {% include 'experiments/components/versions/version_field_details.html' with field=field field_changed=field.changed diff_color='bg-emerald-400' %}
```

Comment:
Do these colors work in light and dark mode? You might need to add `dark:xxx` variants

Reply:
I think it works well in light and dark mode Thougths?:

![image](https://github.com/user-attachments/assets/fa038d29-8cc0-4cbb-bfd9-405574f14ed4)

![image](https://github.com/user-attachments/assets/48fd8be1-a10a-44d2-8ccd-7b2adaa3ef20)

Reply:
Can I propose that we use a slightly more readable dark red text on light red background and dark green on light green? Similar to the badges here:

https://flowbite.com/docs/components/badge/

I'm guessing it'll still work on both, but I think will be a little easier on the eyes

Reply:
Ooh thanks, this is a little easier on the eyes

## Dark mode
![image](https://github.com/user-attachments/assets/4872616f-b594-4c6c-8364-bf46c590254c)

## Light mode
![image](https://github.com/user-attachments/assets/4480b137-1bbd-4d0a-9291-29056a52d447)